app.js

```diff
@@ -1,11 +1,97 @@
 /* Imports */
-
+import { renderMice } from './utils.js';
 /* Get DOM Elements */
+const miceEl = document.getElementById('mice');
+const catLivesEl = document.getElementById('cat-lives');
+const catImgEl = document.getElementById('cat-img');
+const inputEl = document.getElementById('mice-input');
+console.log(inputEl);
+const buttonEl = document.getElementById('mice-button');
+const chasedCountEl = document.getElementById('chased-count');
 
 /* State */
+let chasedCount = 0;
+let catLives = 9;
+const mice = [
+    {
+        name: 'speedy',
+        hp: 5,
+    },
+    {
+        name: 'mousy',
+        hp: 3,
+    },
+    {
+        name: 'tiny',
+        hp: 2,
+    },
+];
 
 /* Events */
+buttonEl.addEventListener('click', () => {
+    const miceName = inputEl.value;
+    console.log(miceName);
+    if (!miceName) {
+        return;
+    }
+    //New mouse =
+    const newMice = {
+        name: miceName,
```

Comment:
we can give the mouse a random name here

Reply:
done!